e2e-playwright/src/page-objects/product-master.po.ts

```diff
@@ -0,0 +1,25 @@
+import { ElementHandleForTag } from 'playwright/types/structs';
+
+export class ProductMasterPage {
+
+  async navigateTo(): Promise<void> {
+    // Could not find a good way to type global jest variables.
+    // See https://stackoverflow.com/questions/64172325/typescript-jest-global-variable-example
+    const url = (global as any).__BASE_URL__;
+    await page.goto(url);
```

Comment:
Wo ist denn `page` definiert? Muss das nicht im Constructor reingereicht werden?

Reply:
Nein, `page` ist nun ein globales Objekt. Hat damit zu tun, dass z.B. Chrome nun nicht mehr hardcoded ist, sondern Jest bestimmt, was wann in welchem Browser läuft. Damit muss Jest den Lifecycle von `page` regeln können.

Reply:
Ah verstehe. Jest sorgt also dafür, dass `page` global bereitsteht.